TestProjects/Windows-Desktop-C#/Console-Application/Console-Application.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net45</TargetFramework>
+    <ProjectGuid>{F15AFA64-1CA5-486A-B9EA-B05377DF2FEC}</ProjectGuid>
```

Comment:
Do we need the `ProjectGuid` tags?

I added them since `sln` changes the `Guid` when a project is removed and re-added. If there's any other reason like Visual Studio needs it, we can keep it, or we can remove them!